homeassistant/components/peco/sensor.py

```diff
@@ -94,9 +94,7 @@ async def async_setup_entry(
 
     async_add_entities(
         [PecoSensor(sensor, county, coordinator) for sensor in SENSOR_LIST],
```

Comment:
We can simplify here and pass a generator expression.
```suggestion
        PecoSensor(sensor, county, coordinator) for sensor in SENSOR_LIST
```